spark/v3.4/spark/src/test/java/org/apache/iceberg/spark/data/TestHelpers.java

```diff
@@ -887,11 +887,14 @@ public static void asMetadataRecord(GenericData.Record file) {
     file.put(3, 0); // specId
   }
 
+  // suppress the readable metrics and first-row-id that are not in manifest files
```

Comment:
```suggestion
  // Suppress the readable metrics and first-row-id fields that are not in manifest files
```

Reply:
I thought we didn't use sentence case in non-javadoc comments because you get weird capitalized fragments and unnecessary PRs/commits to "fix" the case. Are these strong nits?

Reply:
I have no idea what our standard on this is really

// [A-Z]
Has 5422 Hits

// [a-z]
Has 10896 Hits

So we definitely are leaning towards no case